public/assets/sass/abstracts/_mixins.scss

```diff
@@ -30,8 +30,12 @@ $breakpoint arguement choices:
     @if $breakpoint == big-desktop {
         @media only screen and (min-width: 112.5em) { @content };    //1800
     }
+    //special order
+    // @if $breakpoint == navbar {
+    //     @media only screen and (min-width: 42.25em) { @content };    //1800
+    // }
 }
 
 // @include respond(phone) {
-    // code
+    
 // }
```

Comment:
Kindly, remove unneeded snippets. If it is saved for later implementation, then add a TODO comment above the snippet.
```
// TODO: statement for future developers
```